process/block/shardblock.go

```diff
@@ -1398,3 +1408,9 @@ func (sp *shardProcessor) DecodeBlockHeader(dta []byte) data.HeaderHandler {
 
 	return &header
 }
+
+// TransactionCounter returns the object that keeps track of the total nr. of processed txs,
+// and txs processed in this block
+func (sp *shardProcessor) TransactionCounter() *transactionCounter {
```

Comment:
Might remove this, we do not want to be able to use it (yet)

Reply:
done.

Reply:
Not done yet. It is used only in the tests.

Reply:
Was done, but the changes were not pushed yet.